clients/vault/src/system.rs

```diff
@@ -609,6 +608,7 @@ impl VaultService {
 			(
 				"Restart Timer",
 				run(async move {
+					tracing::info!("Periodic restart in 3 hours.");
```

Comment:
Let's insert the actual value of `RESTART_INTERVAL` in the log, otherwise this leads to confusion once we change the constant.